setup.js

```diff
@@ -0,0 +1,70 @@
+
+//azure config mode arm
+var uuid = require('node-uuid');
+var fs = require('fs');
+var spawn = require('child_process').spawn;
+
+var uuidrg = uuid.v4().toLowerCase().replace(/-/g, '');
+var resourceGroup = "rg" + uuidrg.substring(0,19);
+var storageAccountName = "sa" + uuidrg.substring(0,19);
+var location = "westus";
+var errorHasOccurred = false;
+var connectionData = "";
+var wasInArmMode = false;
+
+executeAzureCLI(['config', 'list'], function() {
+	executeAzureCLI(['config', 'mode', 'arm'], function() {
```

Comment:
Are you planning on updating this code soon? Basically, what is the timeline to fix this then?